src/controllers/urls.ts

```diff
@@ -65,19 +65,16 @@ const deleteURL = async (req: Request, res: Response) => {
   }
 };
 
-const __getURLsFromDb = async (tags: string[] | "all" = ["all"], limit: number = 100) => {
-  tags = Array.isArray(tags) ? tags : [tags];
-
+const __getURLsFromDb = async (tags: string[] | undefined, limit: number = 100) => {
   try {
     let query;
-    if (tags.includes("all")) {
+    if (!tags) {
```

Comment:
any reson why "tags.includes("all"))" code and isArray check is removed ?

Reply:
Yep because from what I read if someone includes "all" as a tag name then all urls would be returned instead of those with a tag named "all"